1.	Identify-Basic-Internet-Problems-with-ping.md

```diff
@@ -0,0 +1,113 @@
+<img src="https://media.licdn.com/mpr/mpr/jc/AAEAAQAAAAAAAAfoAAAAJDRmY2I4ZmE1LTk4ZmQtNGUzOS1hOWQ4LTNkMGJkOWQxNjNkYg.png" width="400"  alt="Sonar screen">
+
+## Identify Basic Internet Problems with "ping"
+
+Next time you call your help desk, do you want to wow them with your networking knowledge? Using a command called “ping”, built right into your existing Mac or Windows computer, will help identify basic connection problems. Okay, this might not be enough to “wow” your fellow team members, however they will appreciate that you started the debug process.
+
+**The "ping" command:**
+
+The ping command is a simple way to verify that another computer can receive information from you. The original author, Mike Muuss, actually named the program after the "ping" sound that a submarine sends to detect objects in the water. If an echo of the ping comes back, it means that there is something out there. In fact, "ping" uses a protocol named "Internet Control Message Protocol Echo Request" as part of its underlying software design.
+
+In its simplest form, the ping command provide two valuable pieces of information, whether the message was echoed back ("64 bytes from…") and how long it takes to receive the message back (e.g., time="6.396 ms"). Depending on what type of computer you are using, you may even get a summary containing minimum, maximum, average, and more. The response time is shown in "ms", or millisecond, which is 1/1000th of a second. A response time of 10ms or less is pretty fast, however values are often in the 100ms range. At much above 200ms you’ll probably notice that you have a sluggish connection.
+
+**When all is well:**
+
+This is what my ping response looks like on my Mac OS X computer when everything is working normally here in Malaysia:
+
+```
+MacBook-Pro:~ ajm$ ping Google.com
+PING google.com (216.58.196.46): 56 data bytes
+64 bytes from 216.58.196.46: icmp\_seq=0 ttl=55 time=6.396 ms
+64 bytes from 216.58.196.46: icmp\_seq=1 ttl=55 time=6.368 ms
+64 bytes from 216.58.196.46: icmp\_seq=2 ttl=55 time=26.773 ms
+64 bytes from 216.58.196.46: icmp\_seq=3 ttl=55 time=6.984 ms
+^C
```

Comment:
Mention somewhere below that you need to `Cmd` + `C` to stop - it won't normally stop otherwise.